code/game/area/ai_monitored.dm

```diff
@@ -1,24 +1,29 @@
 /area/ai_monitored
 	name = "AI Monitored Area"
-	var/obj/machinery/camera/motioncamera = null
+	var/list/obj/machinery/camera/motioncameras = list()
+	var/list/motionTargets = list()
 
-
-/area/ai_monitored/New()
+/area/ai_monitored/initialize()
 	..()
-	// locate and store the motioncamera
-	spawn (20) // spawn on a delay to let turfs/objs load

```

Comment:
It was basically a ghetto initialize function. So moving the actions from New to initialize handles things fine. 

I actually had a stack_trace in there that detected monitored areas without motion cameras and it actually detected that the cameras in AI atmos and AI cyborg service are NOT motion cameras. And that was just boxstation. If you want I could add it back in.